server/src/main/java/io/crate/replication/logical/action/PublicationsStateAction.java

```diff
@@ -163,13 +175,41 @@ protected ClusterBlockException checkBlock(Request request,
         }
 
         @VisibleForTesting
-        static List<RelationName> resolveRelationsNames(Publication publication, Schemas schemas) {
+        static User ensurePublicationOwnerExists(String publicationOwner, UserLookup userLookup, String publicationName) {
```

Comment:
Isn't this code obsolete if we ensure that a user owning a publication cannot be dropped?

Reply:
Yep, removed in https://github.com/crate/crate/pull/11872/commits/a8aad52db2c861f766852d52032f3eff448f52f3